interfaces/HO-Portal/src/assets/i18n/en.json

```diff
@@ -457,7 +457,13 @@
       "change-password": {
         "title": "Change password",
         "new-password": "New password",
+        "confirm-password": "Confirm password",
         "success": "Password changed"
+      },
+      "error-messages": {
+        "password-length": "Password should be more than 8 charachters.",
```

Comment:
We should use a variable inside the translations as well.
I think we do that in the PA-app already.

Reply:
Done